ph5/clients/ph5availability.py

```diff
@@ -265,6 +265,116 @@ def get_time_das_t(self, das, start, end,
         self.ph5.forget_das_t(das)
         return earliest_epoch, latest_epoch, new_das_t
 
+    def get_one_availability(self, das, das_info, sample_rate, chan,
+                             start=None, end=None):
```

Comment:
Is an exact match to: https://github.com/PIC-IRIS/PH5/blob/master/ph5/core/ph5api.py#L1410.

I would suggest that we consider removing all references of ph5api.get_availability, but we don't have a good map of all of it's dependent functions to ph5api.get_availability, and so I do not think it is worth prioritizing a dangerous and potentially arduous refactor at this point.